packages/server/README.md

```diff
@@ -86,6 +86,76 @@ Open the URL it prints in a web browser. It will show [Apollo Sandbox](https://w
 
 Apollo Server's built-in Express middleware lets you run your GraphQL server as part of an app built with [Express](https://expressjs.com/), the most popular web framework for Node.
 
+### Using express >= v4.16
+First, install Apollo Server, the JavaScript implementation of the core GraphQL algorithms, Express, and one common Express middleware package:
+
+```
+npm install @apollo/server graphql express cors
+```
+
+If using Typescript you may also need to install additional type declaration packages as development dependencies to avoid common errors when importing the above packages (i.e. Could not find a declaration file for module '`cors`'):
+
+```
+npm install --save-dev @types/cors @types/express
+```
+
+Then, write the following to `server.mjs`. (By using the `.mjs` extension, Node lets you use the `await` keyword at the top level.)
+
+```js
+import { ApolloServer } from '@apollo/server';
+import { expressMiddleware } from '@apollo/server/express4';
+import { ApolloServerPluginDrainHttpServer } from '@apollo/server/plugin/drainHttpServer'
+import express from 'express';
+import http from 'http';
+import cors from 'cors';
+
+// The GraphQL schema
+const typeDefs = `#graphql
+  type Query {
+    hello: String
+  }
+`;
+
+// A map of functions which return data for the schema.
+const resolvers = {
+  Query: {
+    hello: () => 'world',
+  },
+};
+
+const app = express();
+const httpServer = http.createServer(app);
+
+// Set up Apollo Server
+const server = new ApolloServer({
+  typeDefs,
+  resolvers,
+  plugins: [ApolloServerPluginDrainHttpServer({ httpServer })],
+});
+await server.start();
+
+app.use(
+  cors(),
+  express.json(),
+  expressMiddleware(server),
+);
+
+await new Promise((resolve) => httpServer.listen({ port: 4000 }, resolve));
+console.log(`🚀 Server ready at http://localhost:4000`);
+```
+
+Now run your server with:
+
+```
+node server.mjs
+```
+
+Open the URL it prints in a web browser. It will show [Apollo Sandbox](https://www.apollographql.com/docs/studio/explorer/sandbox/), a web-based tool for running GraphQL operations. Try running the operation `query { hello }`!
+
+### Using express < v4.16
```

Comment:
Is this particularly common? Express 4.16 was released in September 2017. The minimum version of Node supported by AS4 (which we should drop because it is no longer in LTS) is 2.5 years newer than Express 4.16.  Is there some sort of challenging backwards-incompatibility in Express 4.16 that makes it common for people to want to use the latest version of Apollo Server but without the ability to use a version of Express from the past 6 years?

In fact, AS4 `@apollo/server` itself depends on `express@^4.17.1`, though admittedly that's only used for the standalone server, not the Express middleware.

Reply:
You're right, but I did want to leave some documentation that suggests the json middleware is required no matter the express version that's being used. Is there any other preferable way to deliever this message?